libr/include/r_io.h

```diff
@@ -3,6 +3,9 @@
 #ifndef R2_IO_H
 #define R2_IO_H
 
+#define _GNU_SOURCE 1
```

Comment:
This produces a lot of warnings while compiling, see https://github.com/radareorg/radare2/pull/16648/checks?check_run_id=622122422#step:7:364

Reply:
This is still no good, it produces a lot of warnings.
Maybe do:
```C
#ifndef _GNU_SOURCE
#define _GNU_SOURCE 1
#endif
```